ontoma/cli.py

```diff
@@ -1,40 +1,71 @@
-import click
 import csv
 import logging
 
+import click
+
+from ontoma.constants import RESULT_FIELDS
 from ontoma.interface import OnToma
-from ontoma.constants import FIELDS
 
 logger = logging.getLogger(__name__)
 
+
 @click.command()
```

Comment:
There isn't a particular reason for using `click`, and indeed everything which it does can be achieved by `argparse`. Before the rewrite this function was already implemented using `click`, so it was easier to keep things as they were.

However, I'm actually starting to think that `click`'s way _might_ be better than that of `argparse`. This annotation of the main entry point function with decorators looks very explicit and readable. Since this is also not a blocker for the release, I've added a discussion point to https://github.com/opentargets/platform/issues/1619.